view/theme/frio/js/theme.js

```diff
@@ -212,23 +212,7 @@ $(document).ready(function(){
 		toggleDropdownText(this);
 	});
 
-	/* setup onoff widgets */
-	// Add the correct class to the switcher according to the input
-	// value (On/Off)
-	$(".toggle input").each(function(){
```

Comment:
I confirm this is still in the current develop, there was something very wrong with your develop merge.